corehq/apps/domain/decorators.py

```diff
@@ -675,3 +675,21 @@ def _wrapped(*args, **kwargs):
         return _wrapped
 
     return _dec
+
+
+def allow_cors(allowed_methods):
```

Comment:
I see the tastypie implementation also verifies that the request method is one of the allowed methods - do you think this should do the same, or does that fall to things like `require_POST` or as in the case api, the body of the view.

Reply:
good call. added this